### PR TITLE
added examples of simple eth transactions

### PR DESCRIPTION
**Pull request by ilmoi**
@tomusdrw

### CODE REVIEW

File: examples/transaction_private.rs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    // Build the tx object
+    let tx_object = TransactionRequest {
+        from,
+        to: Some(to),
+        value: Some(U256::exp10(17)), //0.1 eth
+        ..Default::default()
+    };
+
+    // Send the tx to infura
```
Comment (tomusdrw) [inline examples/transaction_private.rs:27]:
```suggestion
    // Send the tx to localhost
```

**Comment by tomusdrw** [comment]
@ilmoi could you please run `cargo fmt`?

**Comment by ilmoi** [comment]
@tomusdrw done

**Comment by tomusdrw** [comment]
thanks a lot!

**Comment by ilmoi** [comment]
fmt still wasn't happy with the imports so I refactored

**Pull request merged**